config/handles_ir_communities_collections.yml

```diff
@@ -0,0 +1,1145 @@
+handle_ir_communities_collections
+# 10th IIFET Conference, July 10-14, 2000, Corvallis, Oregon, U.S.A.  
+ "30554" : "https://ir.library.oregonstate.edu/catalog?q=10th%20IIFET%20Conference%2C%20July%2010-14%2C%202000%2C%20Corvallis%2C%20Oregon%2C%20U.S.A.&search_field=dspace_community"
```

Comment:
This line should be:
`"30554": "https://ir.library.oregonstate.edu/catalog?q=10th%20IIFET%20Conference%2C%20July%2010-14%2C%202000%2C%20Corvallis%2C%20Oregon%2C%20U.S.A.&search_field=dspace_community"`